openmc/data/data.py

```diff
@@ -218,3 +218,7 @@ def atomic_mass(isotope):
         isotope = isotope[:isotope.find('_')]
 
     return _ATOMIC_MASS.get(isotope.lower())
+
+def kT_to_K(kT):
+    K = kT / 8.6173324e-11
```

Comment:
I'd say just make this value a module variable, `K_BOLTZMANN`, and just get rid of the function.

Also, I just noticed that the value changed in CODATA 2014... hmmmm